### PR TITLE
Computed std name

### PR DESCRIPTION
**Pull request by daltonkell**

### CODE REVIEW

**Comment by benjwadams** [comment]
Please use `cf_1_7_rebase` as the base of your PRs until we pass the CF 1.7 milestone.

File: compliance_checker/cf/cf.py [hunk 1]
```diff
@@ -139,6 +139,109 @@ def setup(self, ds):
         self._find_cf_standard_name_table(ds)
         self._find_geophysical_vars(ds)
 
+    def _check_conventions_version(self, ds):
```
Comment (benjwadams) [inline compliance_checker/cf/cf.py]: Was this intended to be part of this PR?

Comment (daltonkell) [reply compliance_checker/cf/cf.py]: It was not... I'll have to investigate what happened here.

**Comment by daltonkell** [comment]
There are a few conflicting additions; closing in favor of #685.

**Pull request closed without merging**